feature/bgp/transport_tests/bgp_transport_parameters

```diff
@@ -0,0 +1,49 @@
+Summary: 
+ - Validate the ability to configure a different TCP port than the default 179 for a remote peer. This should be tested for a EBGP as well as IBGP neighborship
+ - Ensure that state information on configured transport parameters can be accurately derived.
+
+Topology:
+ATE (Port1) <-IBGP-> (Port1) DUT (Port2) <-EBGP-> (Port2) ATE
+  - Connect ATE Port1 to DUT port1
+  - Connect ATE Port2 to DUT port2
+
+Procedure:
+  - Establish IS-IS adjacency between ATE Port1 and DUT Port1. 
+  - Establish BGP sessions as follows between ATE and DUT . 
+    - ATE port 1 is used for IBGP connection between the Loopback address of the DUT and the IS-IS learnt address behind ATE:Port1. Please ensure that the ATE has BGP listening on a different TCP port than 179 (example: 1800)
+    - The DUT has eBGP peering with ATE port 2 and is receiving IPv4/6 routes. Here too for EBGP, ensure that ATE is using a different transport than default 179 (example: 1800).
+  - Validate session and transport states on ATE and DUT ports using telemetry.
+    - As part of the validation, we should ensure that the values for the leaves "neighbors/neighbor/state/neighbor-port" and "neighbors/neighbor/transport/state/remote-port" are as configured above
+    - Ensure that the remote address derived from the leaves, "neighbors/neighbor/state/neighbor-address" and "neighbor/transport/state/remote-address" are as configured for the neighbors
+  - Validate session state and capabilities received on DUT using telemetry.
+  - Validate the BGP route/path and corresponding attributes for v4 and v6 prefixes
```

Comment:
I think checking that all routes send by ATE on one session are revived by ATE over other session would be suffiicent. Why to complicate this test with attribute validation?

Reply:
We can modify at a later date. But for now I would like to maintain it this way.